src/server/scripts/Kalimdor/CavernsOfTime/CullingOfStratholme/boss_meathook.cpp

```diff
@@ -88,6 +87,7 @@ class boss_meathook : public CreatureScript
             {
                 Talk(SAY_DEATH);
                 _JustDied();
+                instance->SetData(DATA_NOTIFY_DEATH, 1);
```

Comment:
override setbossstate at instance? much cleaner and everything grouped in one place imo